src/examples/operational-apis-demo.ts

```diff
@@ -0,0 +1,88 @@
+import { isEmpty } from 'lodash'
+import { fetchAccessToken, fetchResources } from '../common/koneapi'
+import { fetchEquipmentBasicInformation, fetchEquipmentStatus, fetchServiceOrdersList, fetchSingleServiceOrder } from '../common/operational-api-supporting-functions'
+
+/**
+ * Update these two variables with your own credentials and equipment identifier or set them up as environment variables.
+ */
+const CLIENT_ID: string = process.env.CLIENT_ID || 'YOUR_CLIENT_ID' // eg. 'dcf48ab0-a902-4b52-8c53-1a9aede716e5'
+const CLIENT_SECRET: string = process.env.CLIENT_SECRET || 'YOUR_CLIENT_SECRET' // eg. '31d1329f8344fc12b1a960c8b8e0fc6a22ea7c35774c807a4fcabec4ffc8ae5b'
+
+/**
+ * Check if the needed variables have been defined
+ */
+const checkRequiredVariables = () => {
```

Comment:
This is third identical definition of the checkRequiredVariables. Please use the `validateClientIdAndClientSecret` from `common/koneapi.ts` instead. And you could make the other examples use that too?